src/commands/cmd_config.c

```diff
@@ -95,19 +95,27 @@ void _Config_set(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
 }
 
 int Graph_Config(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
-	// CONFIG <GET|SET> <NAME> [value]
+	// GRAPH.CONFIG <GET|SET> <NAME> [value]
 	if(argc < 3) return RedisModule_WrongArity(ctx);
 
 	const char *action = RedisModule_StringPtrLen(argv[1], NULL);
 
 	if(!strcasecmp(action, "GET")) {
-		// CONFIG GET <NAME>
+		// GRAPH.CONFIG GET <NAME>
 		if(argc != 3) return RedisModule_WrongArity(ctx);
 		_Config_get(ctx, argv, argc);
 	} else if(!strcasecmp(action, "SET")) {
-		// CONFIG SET <NAME> [value]
-		if(argc != 4) return RedisModule_WrongArity(ctx);
-		_Config_set(ctx, argv, argc);
+		// GRAPH.CONFIG SET <NAME> [value] <NAME> [value] ...
+		// emit an error if we received an odd number of arguments,
+		// as this indicates an invalid configuration
+		if(argc < 4 || (argc % 2) == 1) return RedisModule_WrongArity(ctx);
+
+		// set configuration for each requested one
+		for(int i = 2; i < argc; i += 2) {
+			RedisModuleString *key = argv[i];
+			RedisModuleString *val = argv[i+1];
+			_Config_set(ctx, key, val);
```

Comment:
`_Config_set` can no longer reply for each config field being updated,
`GRAPH.CONFIG SET a 1 b 2` should reply with a single response
if the list of configuration contains at least one invalid configuration, the entire command should fail and the configuration should be restored to its previous state.